x-pack/plugins/security_solution/scripts/openapi/bundle.js

```diff
@@ -15,6 +15,20 @@ bundle({
   sourceGlob: join(SECURITY_SOLUTION_ROOT, 'common/api/**/*.schema.yaml'),
   outputFilePath: join(
     SECURITY_SOLUTION_ROOT,
-    'target/openapi/security_solution-{version}.bundled.schema.yaml'
+    'target/openapi/serverless/security_solution-{version}.bundled.schema.yaml'
   ),
+  options: {
+    includeLabels: ['serverless'],
+  },
+});
+
+bundle({
+  sourceGlob: join(SECURITY_SOLUTION_ROOT, 'common/api/**/*.schema.yaml'),
+  outputFilePath: join(
+    SECURITY_SOLUTION_ROOT,
+    'target/openapi/ess/security_solution-{version}.bundled.schema.yaml'
+  ),
+  options: {
+    includeLabels: ['ess'],
+  },
```

Comment:
This generates the following bundles:

```
 succ 📖  Wrote bundled OpenAPI specs to /Users/georgii/Code/elastic/kibana-pr1/x-pack/plugins/security_solution/target/openapi/serverless/security_solution-2023_10_31.bundled.schema.yaml
 succ 📖  Wrote bundled OpenAPI specs to /Users/georgii/Code/elastic/kibana-pr1/x-pack/plugins/security_solution/target/openapi/ess/security_solution-2023_10_31.bundled.schema.yaml
 succ 📖  Wrote bundled OpenAPI specs to /Users/georgii/Code/elastic/kibana-pr1/x-pack/plugins/security_solution/target/openapi/serverless/security_solution-1.bundled.schema.yaml
 succ 📖  Wrote bundled OpenAPI specs to /Users/georgii/Code/elastic/kibana-pr1/x-pack/plugins/security_solution/target/openapi/ess/security_solution-1.bundled.schema.yaml
```

- Do we need bundles for internal endpoints?
- At what point we will filter by `x-internal`?

Reply:
`x-internal` works as expected. The problem is that two Entity Analytics API endpoints described in corresponding OpenAPI specs have paths starting with `/api/` which supposes these are public but don't have `x-internal: true`.

Reply:
Follow-up ticket for the Entity Analytics team: https://github.com/elastic/entity-analytics/issues/60